### DISCUSSION TITLE
Add search and pagination to Time Series Table Chart

### DISCUSSION THREAD

**Discussion by lctdulac** [apache/superset#18440] [Ideas]
Hi SuperSet community!

There is no filter box nor page-like option for Time-Series tables. 

<img width="792" alt="Screenshot 2021-02-10 at 10 37 12" src="https://user-images.githubusercontent.com/46674763/107492439-bb7c2180-6b8c-11eb-991e-a684b6478897.png">

It'd be great if it was possible to do it as in the Table view (below) :

<img width="1429" alt="Screenshot 2021-02-10 at 10 37 28" src="https://user-images.githubusercontent.com/46674763/107492451-bf0fa880-6b8c-11eb-84da-c2cd6f69bc07.png">

What do you guys think ?

Thanks,
Lance

**Comment by junlincc** [comment]
changing title to `add search and pagination to Time Series Table Chart`

Thanks for suggesting! @lctdulac can you let us know your use cases? why/how do you find adding search and pagination helpful in time series table chart?

**Comment by lctdulac** [comment]
Thanks @junlincc for the answer!
My use case is that I am monitoring 200+ time-series which is a bit hard to do in a single page because :
* The header columns disappear when scrolling down
* There is no search bar that can help navigating (like in the table view) without filtering the whole dashboard
* There is no pagination which helps to understand the number of time series there is in the chart.

All in all this is just cosmetics but I think this chart could benefit a lot from these small changes.
Hopefully that makes sense.

**Comment by junlincc** [comment]
@lctdulac thanks for the additional information! It really makes sense from the product standpoint. I wish we can get to it soon😔 Feel free to implement the proposed change though, we are happy to review. 🙏
